SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -403,14 +424,25 @@ public SingularityPendingRequestParent scheduleImmediately(SingularityUser user,
 
     checkConflict(requestWithState.getState() != RequestState.PAUSED, "Request %s is paused. Unable to run now (it must be manually unpaused first)", requestWithState.getRequest().getId());
 
+    // Check these to avoid unnecessary calls to taskManager
+    Integer activeTasks = null;
+    Integer pendingTasks = null;
```

Comment:
I think we want to default these to `0` instead of `null` to avoid NPE's in `checkRunNowRequest()`.

Reply:
Yeah I can do that 👍